Production/crab/configs/2022/DY.yaml

```diff
@@ -0,0 +1,5 @@
+config:
+  params:
+    cmsRunOptions: "sampleType=MC,era=Run3"
+
+DYto2TautoMuTauh: /DYto2TautoMuTauh_M-50_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv3-124X_mcRun3_2022_realistic_v12-v2/MINIAODSIM
```

Comment:
I would also add leptonic inclusive DY, it is stange that now they don't have HT-binned: [/DYTo2L_MLL-50_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv3-124X_mcRun3_2022_realistic_v12-v2/MINIAODSIM](https://cmsweb.cern.ch/das/request?input=dataset%3D%2FDYTo2L_MLL-50_TuneCP5_13p6TeV_pythia8%2FRun3Summer22MiniAODv3-124X_mcRun3_2022_realistic_v12-v2%2FMINIAODSIM&instance=prod/global)